### PR TITLE
Add Changes section

### PR DESCRIPTION
**Pull request by anssiko**
This section summarizes substantial changes and notable editorial improvements to guide review.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/deviceorientation/pull/125.html" title="Last updated on Dec 11, 2023, 12:49 PM UTC (91ab4b5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/deviceorientation/125/e81d70f...91ab4b5.html" title="Last updated on Dec 11, 2023, 12:49 PM UTC (91ab4b5)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -913,3 +913,38 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
...
+- Add Permissions Policy integration
+- Add note to implementers about bundling permission requests
+- Export powerful features Accelerometer, Gyroscope and Magnetometer
+- Add Permissions API integration, start requiring requestPermission() usage
+- editorial: Define API section more normatively and with more dfns
+- editorial: Reorder acceleration explanation in Device Motion Model section
+- editorial: Update explanations of the device rotation and motion refences
+- editorial: Use more precise event handling terms, modernize others
+- editorial: Refer to [[SCREEN-ORIENTATION]] instead of the orientation change event
+- editorial: Reword requirements in "Security and privacy considerations
```
Comment (reillyeon) [inline index.bs:931]:
```suggestion
- editorial: Reword requirements in "Security and privacy considerations"
```

File: index.bs [hunk 2]
```diff
@@ -913,3 +913,38 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
...
+
+This section summarizes substantial changes and notable editorial improvements to guide review. Full details are available from the <a href="https://github.com/w3c/deviceorientation/commits/main">commit log</a>. Changes since the <a href="https://www.w3.org/TR/2016/CR-orientation-event-20160818/">Candidate Recommendation 2016-08-18</a>:
+
+- Add Permissions Policy integration
+- Add note to implementers about bundling permission requests
+- Export powerful features Accelerometer, Gyroscope and Magnetometer
+- Add Permissions API integration, start requiring requestPermission() usage
+- editorial: Define API section more normatively and with more dfns
+- editorial: Reorder acceleration explanation in Device Motion Model section
+- editorial: Update explanations of the device rotation and motion refences
```
Comment (reillyeon) [inline index.bs:928]:
```suggestion
- editorial: Update explanations of the device rotation and motion references
```

File: index.bs [hunk 3]
```diff
@@ -913,3 +913,38 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
...
+- Update IDL dictionaries with new dictionary defaulting setup
+- Note the deviceorientationabsolute event and its ondeviceorientationabsolute event handler IDL attribute have limited implementation experience
+- Add requestPermission() API static operation to both DeviceOrientationEvent and DeviceMotionEvent
+- Add [SecureContext] to event handlers ondeviceorientation, ondevicemotion and ondeviceorientationabsolute
+- Restrict all interfaces to secure contexts
+- Remove [NoInterfaceObject] from DeviceAcceleration and DeviceRotationRate
+- Make security and privacy considerations normative
+- Add the ondeviceorientationabsolute event handler attribute into the IDL block (was only in prose)
+- Remove '?' from dictionary members of DeviceMotionEventInit
+- Use [Exposed=Window] extended attribute
```
Comment (reillyeon) [inline index.bs:950]:
```suggestion
- Use [Exposed=Window] extended attribute

```

**Review by reillyeon — APPROVED** [review]
LGTM with editorial nits

**Comment by anssiko** [comment]
Thanks @reillyeon. Knowing @rakuco is transitioning into a well-deserved vacation mode, I will bravely merge this to make his review backlog a tad shorter when he returns. Any substantive changes added should be reflected to this section to aid reviewers.

**Pull request merged**